tests/test_create_climo_files.py

```diff
@@ -505,6 +512,12 @@ def test_resolution_warning(period, outdir, tiny_dataset):
             convert_longitudes=True, output_resolutions={'monthly'}
         )
 
+    def check_warnings(record):
```

Comment:
Just one small comment: I'd name this function as a yes/no question (e.g. "contains_warning"). Without reading the function, it's not clear what the "check" is doing.

Otherwise the PR looks good to me! Good call on the "slow" marking. Man, are some of these tests slow!

```
============================================= 338 passed, 331443 warnings in 1278.69s (0:21:18) =============================================
```